src/vellum/workflows/nodes/displayable/subworkflow_deployment_node/node.py

```diff
@@ -252,44 +274,68 @@ def run(self) -> Iterator[BaseOutput]:
-                        yield BaseOutput(
-                            name=event.data.output.name,
-                            delta=event.data.output.delta,
-                        )
-                    elif event.data.output.state == "FULFILLED":
-                        yield BaseOutput(
-                            name=event.data.output.name,
-                            value=event.data.output.value,
+
+        try:
+            for event in subworkflow_stream:
+                if event.type != "WORKFLOW":
+                    continue
+                if event.data.state == "INITIATED":
+                    continue
+                elif event.data.state == "STREAMING":
+                    if event.data.output:
+                        if event.data.output.state == "STREAMING":
+                            yield BaseOutput(
+                                name=event.data.output.name,
+                                delta=event.data.output.delta,
+                            )
+                        elif event.data.output.state == "FULFILLED":
+                            yield BaseOutput(
+                                name=event.data.output.name,
+                                value=event.data.output.value,
+                            )
+                            fulfilled_output_names.add(event.data.output.name)
+                elif event.data.state == "FULFILLED":
+                    outputs = event.data.outputs
+                elif event.data.state == "REJECTED":
+                    error = event.data.error
+                    if not error:
+                        raise NodeException(
+                            message="Expected to receive an error from REJECTED event",
+                            code=WorkflowErrorCode.INTERNAL_ERROR,
                         )
-                        fulfilled_output_names.add(event.data.output.name)
-            elif event.data.state == "FULFILLED":
-                outputs = event.data.outputs
-            elif event.data.state == "REJECTED":
-                error = event.data.error
-                if not error:
-                    raise NodeException(
-                        message="Expected to receive an error from REJECTED event",
-                        code=WorkflowErrorCode.INTERNAL_ERROR,
-                    )
-                workflow_error = workflow_event_error_to_workflow_error(error)
-                raise NodeException.of(workflow_error)
+                    workflow_error = workflow_event_error_to_workflow_error(error)
+                    raise NodeException.of(workflow_error)
+        except WorkflowInitializationException as e:
+            if not release_id:
+                release_id = self._get_release_id(deployment_name, self.release_tag)
+
+            raise NodeException(
+                message=e.message,
+                code=e.code,
+                raw_data={"release_id": release_id} if release_id else None,
+            ) from e
```

Comment:
In the API version, we should just pass through the raw data and stack trace from event.data.error. we don't expect a `WorkflowInitializationException` here